recipes/anypytools/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.9.0" %}
+
+package:
+  name: anypytools
+  version: {{ version }}
+
+source:
+   fn: {{ version }}.tar.gz
+   url: https://github.com/AnyBody-Research-Group/AnyPyTools/archive/{{ version }}.tar.gz
+   sha1: e7d02a0c0edbc2729356343526677e2b3e283381
+
+build:          
+  number: 0
+  # Added to allow the pytest entry points in setup.py to work
+  # This is also the reason for setuptools beeing a runtime dependency. 
+  preserve_egg_dir: True
+  skip: True  # [not win]
+
+requirements:
+  build:
+    - setuptools
+    - python
+    - future
+
+  run:
+    - python
+    - pywin32  # [py27 or py34]
+    - future >=0.15
+    - h5py >=2.5
+    - setuptools
+    - scipy >=0.15
+    - jupyter >=1.0
+    - ipython >=4.0
+
+test:
+    imports:
+       - anypytools  # [win]
+       - anypytools.abcutils  # [win]
+       - anypytools.macro_commands  # [win]
```

Comment:
Sorry trying to get over a cold and clearly not thinking correctly. Could we please remove these selectors?